pyface/ui/qt4/tests/test_qt_imports.py

```diff
@@ -21,10 +22,13 @@ def test_imports(self):
         import pyface.qt.QtOpenGL  # noqa: F401
         import pyface.qt.QtSvg  # noqa: F401
         import pyface.qt.QtTest  # noqa: F401
-        import pyface.qt.QtWebKit  # noqa: F401
         import pyface.qt.QtMultimedia  # noqa: F401
         import pyface.qt.QtMultimediaWidgets  # noqa: F401
 
+    @unittest.skipIf(sys.version_info > (3, 6), "Web engine not supported yet")
```

Comment:
I think that "WebKit not available" is more informative.

Reply:
```suggestion
    @unittest.skipIf(sys.version_info > (3, 6), "WebKit is not available")
```